records_mover/records/mover.py

```diff
@@ -117,8 +117,7 @@ def move(records_source: RecordsSource,
     elif isinstance(records_source, SupportsToFileobjsSource):
         # Incompatible types in assignment (expression has type "Optional[Any]",
         #     variable has type "BaseRecordsFormat")
-        target_records_format: BaseRecordsFormat = getattr(records_target,
-                                                           "records_format", None)  # type: ignore
+        target_records_format: BaseRecordsFormat = getattr(records_target, "records_format", None)  # type: ignore
```

Comment:
E501 line too long (114 > 100 characters)

Reply:
I agree, this line is too long

Reply:
oh no, I've become the BlueLabs Butler.